.github/workflows/security-scan.yml

```diff
@@ -6,10 +6,11 @@ on:
 
 jobs:
   security_scan:
-    uses: cccteam/github-workflows/.github/workflows/golang-security-scan.yml@v2.0.2
+    uses: cccteam/github-workflows/.github/workflows/golang-security-scan.yml@v3.0.0
     with:
       go-version: "1.22.3"
       execute-trivy-scan: false
     secrets:
       SLACK_BOT_TOKEN: ${{ secrets.CCC_SLACK_BOT_TOKEN }}
       SLACK_CHANNEL_ID: ${{ secrets.SLACK_CHANNEL_ID }}
+      DEPENDABOT_ALERTS_TOKEN: ${{ secrets.SECURITY_ALERTS_TOKEN }}
```

Comment:
You had said you wanted to discuss aligning `DEPENDABOT_ALERTS_TOKEN` vs `secrets.SECURITY_ALERTS_TOKEN`... If I remember correctly, this is a PAT that only has the **_security_events_** scope, so maybe we should align both names to `SECURITY_EVENTS_TOKEN`. Thoughts?

Reply:
This would require another "breaking change" release for the **_github-workflows_** project.

Reply:
If we stuck with `DEPENDABOT_ALERTS_TOKEN`, it would not require a breaking change to **github-workflows**. Using `DEPENDABOT_ALERTS_TOKEN` seems more clear on what it is for.

Reply:
Yeah, that would be completely fine with me.

Reply:
@jwatson-ccc Was this addressed?

Reply:
The `DEPENDABOT_ALERTS_TOKEN` has been put out there. I don't think I triggered a Security Scan to validate all works.